src/features/user/api/delete-post.ts

```diff
@@ -0,0 +1,14 @@
+export const deletePost = (post_id: number) => {
+    const response: any = fetch(`http://localhost:3000/delete-post/${post_id}`,{
```

Comment:
bez anyyyy

Reply:
No ale wyświetla mi się błąd ze response nie ma typu : (

Reply:
A to jest duży obiekt

Reply:
Dobra, jednak mam jakiś typ, nevermind